contracts/base/ERC721Consumable.sol

```diff
@@ -0,0 +1,71 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+// import "@openzeppelin/contracts/utils/Counters.sol";
+import "./IERC721Consumable.sol";
+
+abstract contract ERC721Consumable is IERC721Consumable, ERC721 {
+    // Mintable
+    //using Counters for Counters.Counter;
+    //Counters.Counter private _tokenIds;
+
+    // Mapping from token ID to consumer address
+    mapping(uint256 => address) _tokenConsumers;
+
+    // // Do we need constructor and mint ?
```

Comment:
Lets remove these

Reply:
Aren't we breaking the consumerOf and changeConsumer() functions if we remove the _tokenConsumers mapping ?

Reply:
Done.